lib/cocoapods-core/podfile/dsl.rb

```diff
@@ -930,6 +930,27 @@ def post_install(&block)
         raise Informative, 'Specifying multiple `post_install` hooks is unsupported.' if @post_install_callback
         @post_install_callback = block
       end
+
+      # This hook allows you to make any last changes to the generated Xcode
+      # project before it is written to disk, or any other tasks you might want
```

Comment:
I think thats not true and at this point this is executed the projects have been saved to disk.

Reply:
Sorry, I have copy paste the post_install method and forgot to update the comment lol, updated :)